src/ron_enc.rs

```diff
@@ -0,0 +1,21 @@
+use serde::Serialize;
+use serde::de::DeserializeOwned;
+use ron::{ser, de};
+
+pub type Repr = String;
+pub type SerializeError = ser::Error;
+pub type DeserializeError = de::Error;
+
+pub fn serialize<T>(value: &T) -> ser::Result<String>
+    where T: Serialize
+{
+    ser::to_string(value)
```

Comment:
I think it would be better to use the `pretty` version of ron rather the minified one, what do you think?

Reply:
given bincode is an option and ron formatters are rare or nonexistent i think using pretty output very much makes sense, i'll change it

Reply:
not quite pretty but better than nothing
```ron
{
    "Lapfox": "\"lapfoxtrax.com\"",
    "point": "(
    x: 2,
    y: 3,
)",
    "Rust": "\"github.com/rust-lang/rust\"",
}
```

Reply:
PR updated